### ISSUE TITLE
Using zoom and can no longer scroll; to update

### ISSUE THREAD

**Issue by evcon131** [rstudio/rstudio#7767]
I have zoom set to the highest level and can no longer change anything in preferences since it is zoomed in and I cant scroll.
![image](https://user-images.githubusercontent.com/32807283/92790700-3f757780-f369-11ea-9e92-9f6b273919de.png)

**Comment by gtritchie** [comment]
What version of RStudio are you using?

On RStudio 1.3 and newer you should be able to scroll the rest of the dialog into view using scrollwheel and/or using the Tab key to move between the controls.

**Comment by gtritchie** [comment]
Fixed in 1.3 via: #5597

**Comment by evcon131** [comment]
Thanks just had to update, thanks!
